seisbench/models/phasenet.py

```diff
@@ -141,11 +156,24 @@ def _merge_skip(skip, x):
         return torch.cat([skip, x_resize], dim=1)
 
     def annotate_window_pre(self, window, argdict):
-        # Add a demean and normalize step to the preprocessing
+        # Add a demean and an amplitude normalization step to the preprocessing
         window = window - np.mean(window, axis=-1, keepdims=True)
-        std = np.std(window, axis=-1, keepdims=True)
-        std[std == 0] = 1  # Avoid NaN errors
-        window = window / std
+        if self.norm_detrend:
+            detrended = np.zeros(window.shape)
+            for i, a in enumerate(window):
+                detrended[i, :] = scipy.signal.detrend(a)
+            window = detrended
+        if self.norm_amp_per_comp:
+            amp_normed = np.zeros(window.shape)
+            for i, a in enumerate(window):
+                amp = a / (np.max(np.abs(a)) + 1e-10)
+                amp[amp == 0] = 1  # Avoid NaN errors
```

Comment:
I think this is a bug that will introduce random spikes. Could you check this? Probably the same for EQT.

Reply:
Thank you for this hint! Of course, that line shouldn't be there. Fortunately, EQT is not concerned.